android/app/src/main/java/ai/fuzzylabs/wearablemyfoot/MainActivity.kt

```diff
@@ -149,13 +162,17 @@ class MainActivity : AppCompatActivity() {
                     sessionService?.reset()
                 }
                 BluetoothLeService.ACTION_GATT_DISCONNECTED -> {
-                    finish()
+//                    finish()
```

Comment:
Why commented out?

Reply:
It shouldn't be. Was trying to find a bug (dealing with concurrency). Uncommented now.